service/gradle/testing.gradle

```diff
@@ -43,13 +43,19 @@ test {
     }
     failOnPassedAfterRetry = true
   }
+  // We don't want to use up all the cores on a developer's machine because it locks them out of doing other
```

Comment:
This is defined in 5 places. Seems like either 1 can be deleted, or 2/3/4/5 can be deleted.